Description: Help md format fixes
 These changes are useful to make a nicer man page
Forwarded: not-needed

--- a/cli/help.md
+++ b/cli/help.md
@@ -1,9 +1,16 @@
 rollup version __VERSION__
 =====================================
 
-Usage: rollup [options] <entry file>
+rollup(1) -- module bundler for JavaScript
+=========================================================
 
-Basic options:
+SYNOPSIS
+--------
+
+rollup [options] <entry file>
+
+OPTIONS
+-------
 
 -c, --config <filename>     Use this config file (if argument is used but value
                               is unspecified, defaults to rollup.config.js)
@@ -74,24 +81,34 @@
 --watch.include <files>     Limit watching to specified files
 --validate                  Validate output
 
-Examples:
+EXAMPLES
+--------
 
-# use settings in config file
+* use settings in config file
+```
 rollup -c
+```
 
-# in config file, process.env.INCLUDE_DEPS === 'true'
-# and process.env.BUILD === 'production'
+* in config file, process.env.INCLUDE_DEPS === 'true'
+and process.env.BUILD === 'production'
+```
 rollup -c --environment INCLUDE_DEPS,BUILD:production
+```
 
-# create CommonJS bundle.js from src/main.js
+* create CommonJS bundle.js from src/main.js
+```
 rollup --format=cjs --file=bundle.js -- src/main.js
+```
 
-# create self-executing IIFE using `window.jQuery`
-# and `window._` as external globals
+* create self-executing IIFE using `window.jQuery`
+and `window._` as external globals
+```
 rollup -f iife --globals jquery:jQuery,lodash:_ \
   -i src/app.js -o build/app.js -m build/app.js.map
+```
 
-Notes:
+NOTES
+-----
 
 * When piping to stdout, only inline sourcemaps are permitted
 
